molgenis-core-ui/package.json

```diff
@@ -0,0 +1,86 @@
+{
+  "name": "MOLGENIS",
+  "version": "1.16.0-SNAPSHOT",
```

Comment:
Can we use Maven process-resources to replace a version token with the current version?